src/lib/stream/rc4/rc4.cpp

```diff
@@ -111,6 +111,6 @@ void RC4::clear()
 /*
 * RC4 Constructor
 */
-RC4::RC4(size_t s) : SKIP(s), X(0), Y(0) {}
+RC4::RC4(size_t s) : SKIP(s), X(0), Y(0), state(), buffer(), position(0) {}
```

Comment:
You're right. Explicit initialization of the vectors is not necessary. Moved initialization of `X`, `Y` and `position` to the header.